tools/mqtt-bench/README.md

```diff
@@ -1,80 +1,106 @@
-# MQTT benchmarking tool
+# MQTT Benchmarking Tool
 
-A simple MQTT (broker) benchmarking tool for Mainflux platform. ( based on github.com/krylovsk/mqtt-benchmark )
+A simple MQTT benchmarking tool for Mainflux platform
+(based on [@krylovsk](https://github.com/krylovsk)'s [mqtt-benchmark](github.com/krylovsk/mqtt-benchmark)).
 
 
+## Installation
+```
+cd tools/mqtt-bench
+make
+```
+
+## Usage
 The tool supports multiple concurrent clients, publishers and subscribers configurable message size, etc:
 
 ```
-cd benchmark
-go build  -o mqtt-bench *.go
+drasko@Marx:~/go/src/github.com/mainflux/mainflux/tools/mqtt-bench$ ./mqtt-bench --help
```

Comment:
Remove your prompt. If really want the path to be there, exchange it with `cd $GOPATH/mainflux/mainflux...` and then the help command